seacrowd/sea_datasets/bhinneka_korpus/bhinneka_korpus.py

```diff
@@ -0,0 +1,181 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+from seacrowd.utils import schemas
+
+_CITATION = """\
+@article{,
+  author    = {},
+  title     = {},
+  journal   = {},
+  volume    = {},
+  year      = {},
+  url       = {},
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
+"""
+
+_DATASETNAME = "bhinneka_korpus"
+_DESCRIPTION = """The Bhinneka Korpus dataset was parallel dataset for five Indonesian Local Languages conducted 
+through a volunteer-driven translation strategy, encompassing sentences in the Indonesian-English pairs and lexical 
+terms. The dataset consist of parallel data with 16,000 sentences in total, details with 4,000 sentence pairs for two 
+Indonesia local language and approximately 3,000 sentences for other languages, and one lexicon dataset creation for 
+Beaye language. In addition, since beaye is a undocumented language, we don't have any information yet about the use 
+of language code. Therefore, we used "day" (a code for land dayak language family) to represent the language."""
+
+_HOMEPAGE = "https://github.com/joanitolopo/bhinneka-korpus"
+_LICENSE = Licenses.APACHE_2_0.value
+_URLS = "https://raw.githubusercontent.com/joanitolopo/bhinneka-korpus/main/"
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGUAGES = ["abs", "day", "mkn", "aoz", "mak"]
+LANGUAGES_MAP = {
+    "abs": "ambonese-malay",
+    "day": "beaye",
+    "mkn": "kupang-malay",
+    "aoz": "uab-meto",
+    "mak": "makassarese"
+}
+
+
+def process_list(data):
```

Comment:
please give the function proper typings if you decide to keep this, but generally I think we don't want a function which can return either a list or a string.